brave/src/main/java/com/linecorp/armeria/client/brave/ArmeriaHttpClientParser.java

```diff
@@ -53,8 +55,10 @@ static ArmeriaHttpClientParser get() {
     private ArmeriaHttpClientParser() {}
 
     @Override
-    public <T> void request(HttpAdapter<T, ?> rawAdapter, T req, SpanCustomizer customizer) {
-        super.request(rawAdapter, req, customizer);
+    public void parse(brave.http.HttpRequest request, TraceContext context, SpanCustomizer span) {
```

Comment:
nit: `span` could be renamed back to `customizer` or `spanCustomizer`?

Reply:
Personally good with span, `Span`, `MutableSpan`, and `SpanCustomizer` are basically the same concept.

Reply:
OK. No need to rename, then. :-)

Reply:
yeah we fixed the name to span upstream. naming parameter based on type was laborious